knowledge_repo/app/routes/debug.py

```diff
@@ -77,7 +77,7 @@ def show_views():
 
         methods = ','.join(rule.methods)
         url = url_for(rule.endpoint, **options)
-        line = urllib.unquote("{:50s} {:20s} {}".format(rule.endpoint, methods, url))
+        line = urllib.parse.unquote("{:50s} {:20s} {}".format(rule.endpoint, methods, url))
```

Comment:
Maybe use the logic below in terms of importing this.

Reply:
@john-bodley sorry can you clarify? Would it be better to `from urllib.parse import unquote` at the top of the file?

Reply:
Ah I see what you were referring to. Best to keep consistent 👍